src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -195,6 +263,22 @@ private static void StartBridge(CommandLineArguments commandLineArgs)
                 Environment.Exit(1);
             }
 
+            string resourceFolder = commandLineArgs.BridgeConfiguration.BridgeResourceFolder;
+            if (String.IsNullOrWhiteSpace(resourceFolder))
+            {
+                Console.WriteLine("Starting the Bridge requires the BridgeResourceFolder to be specified.");
+                Console.WriteLine("Use either -BridgeResourceFolder:folderName or set it as an environment variable.");
+                Environment.Exit(1);
```

Comment:
Good idea.  How about doing this as soon as we return anything other than 0 or 1?  I don't think it needs a separate issue for tracking but will do it as soon as we need another exit code.